cmd/doracled/cmd/root.go

```diff
@@ -55,7 +55,7 @@ func init() {
 		genOracleKeyCmd,
```

Comment:
Since we are changing this to a function, it would be good to change all other Cmd to a function.

Reply:
Thanks, I change all other Cmd to function form in [f7b488e](https://github.com/medibloc/panacea-doracle/pull/46/commits/f7b488e1dd94f4dfc16eac4e9b5036b9ba09d41c).